concepts/40 Angular Components/40 Common Features/10 Tree Shaking.md

```diff
@@ -1,6 +1,8 @@
-Tree shaking removes unused portions of source and library code to reduce the downloaded size of the application.
+## Tree Shaking
 
-Tree shaking already works if you import only the modules that you use in your application. For instance, the following import takes only the `DxButtonModule` from the `devextreme-angular` module:
+The tree shaking mechanism removes unused portions of source and library code to reduce the downloaded size of the application.
+
+The tree shaking mechanism checks whether you import only the modules used in your application. For instance, the following import takes the `DxButtonModule` from the `devextreme-angular` module:
```

Comment:
Since we already mention "The tree shaking mechanism" in line 3 above, can we change the phrase to "This mechanism" on this line?

`This mechanism checks whether you import only the modules used in your application. For instance, the following **import** takes the `DxButtonModule` from the `devextreme-angular` module:`